mne/utils/docs.py

```diff
@@ -261,814 +175,589 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
+    plotted. Bad channels will be excluded from these calculations.
+    If None, no area will be plotted. If average=False, no area is plotted.
 """
-docdict['annot_ch_names'] = """
-ch_names : list | None
-    List of lists of channel names associated with the annotations.
-    Empty entries are assumed to be associated with no specific channel,
-    i.e., with all channels or with the time slice itself. None (default) is
-    the same as passing all empty lists. For example, this creates three
-    annotations, associating the first with the time interval itself, the
-    second with two channels, and the third with a single channel::
 
-        Annotations(onset=[0, 3, 10], duration=[1, 0.25, 0.5],
-                    description=['Start', 'BAD_flux', 'BAD_noise'],
-                    ch_names=[[], ['MEG0111', 'MEG2563'], ['MEG1443']])
+docdict['aseg'] = """
+aseg : str
+    The anatomical segmentation file. Default ``aparc+aseg``. This may
+    be any anatomical segmentation file in the mri subdirectory of the
+    Freesurfer subject directory.
 """
 
-# General plotting
-docdict["show"] = """
-show : bool
-    Show the figure if ``True``.
-"""
-docdict["title_None"] = """
-title : str | None
-    The title of the generated figure. If ``None`` (default), no title is
-    displayed.
+docdict['average_psd'] = """
+average : str | None
+    How to average the segments. If ``mean`` (default), calculate the
+    arithmetic mean. If ``median``, calculate the median, corrected for
+    its bias relative to the mean. If ``None``, returns the unaggregated
+    segments.
 """
-docdict["plot_proj"] = """
-proj : bool | 'interactive' | 'reconstruct'
-    If true SSP projections are applied before display. If 'interactive',
-    a check box for reversible selection of SSP projection vectors will
-    be shown. If 'reconstruct', projection vectors will be applied and then
-    M/EEG data will be reconstructed via field mapping to reduce the signal
-    bias caused by projection.
 
-    .. versionchanged:: 0.21
-       Support for 'reconstruct' was added.
-"""
-docdict["evoked_topomap_ch_type"] = """
-ch_type : 'mag' | 'grad' | 'planar1' | 'planar2' | 'eeg' | None
-    The channel type to plot. For 'grad', the gradiometers are collected in
-    pairs and the RMS for each pair is plotted.
-    If None, then channels are chosen in the order given above.
-"""
-docdict["topomap_vmin_vmax"] = """
-vmin, vmax : float | callable | None
-    Lower and upper bounds of the colormap, in the same units as the data.
-    If ``vmin`` and ``vmax`` are both ``None``, they are set at ± the
-    maximum absolute value of the data (yielding a colormap with midpoint
-    at 0). If only one of ``vmin``, ``vmax`` is ``None``, will use
-    ``min(data)`` or ``max(data)``, respectively. If callable, should
-    accept a :class:`NumPy array <numpy.ndarray>` of data and return a
-    float.
+docdict['average_plot_psd'] = """
```

Comment:
My test appears to have been useful:
```
E   AssertionError: assert ['accept', 'a...clust_1', ...] == ['accept', 'a...clust_1', ...]
E     At index 20 diff: 'average_plot_psd' != 'average_psd'
E     Use -v to get the full diff
```
Should I push and you can iterate to fix these?